docker-compose.yml

```diff
@@ -16,13 +16,13 @@ services:
     build:
       context: .
       dockerfile: docker/Dockerfile.amd64
-      # dockerfile: docker/Dockerfile.core.dev
+       #dockerfile: docker/Dockerfile.core.dev
     devices:
       - /dev/bus/usb:/dev/bus/usb
       - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
     volumes:
       - /etc/localtime:/etc/localtime:ro
-      - ./config/config.yml:/config/config.yml:ro
+      - ./config/:/config/:ro
```

Comment:
undo?

Reply:
So with my testing of the dev container on windows and mac, didn't matter. This didn't work. It would make a directory called config.yml and break Frigate. It wouldn't actually copy the config file over, it would just have a corrupted directory. If you know the fix let me know. I am happy to revert since this was just to make dev work for me.